### DISCUSSION TITLE
Account Balance is Hidden on mobile device

### DISCUSSION THREAD

**Discussion by Hohenbostler** [firefly-iii/firefly-iii#6794] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [ ] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [ ] ... there are no open or closed issues that are related to my problem
- [ ] ... it's [definitely me, not you](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

Hi. 

at First sorry for my Bad english ;)

I have the Problem the it cant See the. Account Balance Like in the normal Version. 
![IMG_20230102_182457](https://user-images.githubusercontent.com/42488069/210263174-a747e9da-e8b1-4d72-811f-86d7caae479e.jpg)

### Expected behaviour

I would See the sum of my Account Balance s 

### Debug information

I did Not make a debug

**Comment by JC5** [comment]
Correct, for some things there has not been made room. This will not be changed, unfortunately.
